src/compute-client/src/response.proto

```diff
@@ -69,7 +69,18 @@ message ProtoSubscribeBatch {
         int64 diff = 3;
     }
 
+    message ProtoSubscribeBatchContents {
+        oneof kind {
+            ProtoSubscribeUpdates updates = 1;
+            string error = 2;
+        }
+    }
+
+    message ProtoSubscribeUpdates {
+        repeated ProtoUpdate updates = 1;
+    }
+
     mz_repr.antichain.ProtoU64Antichain lower = 1;
     mz_repr.antichain.ProtoU64Antichain upper = 2;
-    repeated ProtoUpdate updates = 3;
+    ProtoSubscribeBatchContents updates = 3;
```

Comment:
Because of our current backward compat requirements, I think we have to reserve the old id and bump the new field:

```suggestion
    reserved 3;
    ProtoSubscribeBatchContents updates = 4;
```

And if this is ever used in json, also need `reserved "updates"` (requiring a new name for the new field).

Reply:
I'm pretty sure this isn't used in any durable way at the moment, though if I misunderstand the significance of your JSON comment I'm happy to try and fix this up.

Reply:
The JSON comment only applies if we serialize this type to JSON and back, because the same name is used for a type that changed, so can disregard that.

> this isn't used in any durable way at the moment

I'm not the person to sign off on if it's ok or not to re-use a protobuf id. We were ok doing it before because we wiped all envs. Doing it after our "no more wipes" release should in general be not done? If we're wrong then things will either error or be silently corrupted.